src/module/apps/effects-panel.ts

```diff
@@ -62,6 +62,7 @@ export class EffectsPanel extends Application {
             actor,
             effects,
             conditions,
+            isGM: game.user.isGM,
```

Comment:
I notice that actor sheets have user: { isGM } instead. (from using the debugger real quick). It might be best to imitate however the actor sheets inject that data in so that the templates more or less match.

Reply:
Changed to that.